includes/class-bsf-custom-fonts-posttype.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Bsf Custom Fonts Taxonomy
+ *
+ * @since  1.0.0
+ * @package Bsf_Custom_Fonts
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit();
+}
+
+if ( ! class_exists( 'BSF_Custom_Fonts_Posttype' ) ) :
+
+	/**
+	 * BSF_Custom_Fonts_Posttype
+	 */
+	class BSF_Custom_Fonts_Posttype {
+		/**
+		 * Instance of BSF_Custom_Fonts_Posttype
+		 *
+		 * @since  1.0.0
```

Comment:
Please change the since to x.x.x or the proper version if this one is invalid. Do this for all instances in this file @imnavanath  if required

Reply:
DONE